src/session/content/chat_history.rs

```diff
@@ -104,9 +120,27 @@ mod imp {
             self.parent_constructed(obj);
             obj.setup_expressions();
 
+            obj.set_sticky(true);
```

Comment:
```suggestion
            self.sticky.set(true);
```
So that we can avoid the unneeded notify.

Reply:
It seems the notification is needed? Otherwise, the scroll button is displayed in the first opened chat, although we are still at the end of the history. This is because glib initializes the property with `false`. 

I don't understand why glib ignores the `true` in
```rust
glib::ParamSpecBoolean::new(
    "sticky",
    "Sticky",
    "Whether the chat history should stick to the newest message",
    true,
    glib::ParamFlags::READABLE,
),
```

Reply:
In this case I think you can just manually implement Default for the private struct of ChatHistory and initialize sticky to true.